src/main/java/com/puppycrawl/tools/checkstyle/checks/imports/CustomImportOrderCheck.java

```diff
@@ -756,21 +773,23 @@ private static int compareImports(String import1, String import2) {
     }
 
     /**
-     * Counts empty lines before given.
-     * @param lineNo
+     * Counts empty lines between given.
```

Comment:
`between given` => `between given parameters`.

Reply:
Done!

Reply:
This was not done.

Reply:
Sorry about missed that :(